x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_rule_actions/schema.tsx

```diff
@@ -6,10 +6,70 @@
 
 import { i18n } from '@kbn/i18n';
 
-import { FormSchema } from '../../../../../shared_imports';
+import {
+  ActionTypeModel,
+  AlertAction,
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
+} from '../../../../../../../../../plugins/triggers_actions_ui/public/types';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { TypeRegistry } from '../../../../../../../../../plugins/triggers_actions_ui/public/application/type_registry';
```

Comment:
Same as the other `import` of these types -- adding a comment here linking to an Alerting issue requesting these types be exposed will be helpful to future developers.